docs/reports-migrating-from-cds-to-li.md

```diff
@@ -93,8 +93,8 @@ This doc compares a sample Corona Data Scraper (CDS) record with a Li record.  _
 ```
 {
   "locationID": "iso1:us#iso2:us-ca#fips:06007",
-  "slug": "butte-county-california-us",
-  "name": "Butte County, California, US",
+  "slug": "butte-county-california-united-states",
```

Comment:
Hm, "Note that this patch changes all location slugs as a side effect." - I believe that this could be a big deal, b/c the location slugs are used in the data model.  I had proposed changing the slugs in https://github.com/covidatlas/li/issues/227, but @ryanblock had thought that it might have repercussions.  I didn't look further into it at the time, or discuss what those implications would be.

@joliss - is the intent of this to improve the generated reports?  If yes, maybe we can change the report generation code only, rather than introduce a change to the slugs as well.

Reply:
Good point -- I wasn't sure if we wanted to have the slugs always match the names, or if we want to keep the current slugs intact.

I pushed a new version of this pull request that keeps the slugs intact. Do you want to take a look again?

Reply:
To the best of my recollection, this would be a massive breaking change. Strongly advise against.